package.json

```diff
@@ -82,7 +86,11 @@
     "tslib": "^2.4.1",
     "typedoc": "^0.23.15",
     "typescript": "^4.9.3",
-    "vue": "^2.7.14",
+    "vue": "^2.7.14"
+  },
+  "dependencies": {
+    "dockerode": "^3.3.4",
+    "fast-xml-parser": "^4.1.1",
```

Comment:
`peerDependencies` ?

Reply:
I do not think we should expect every project that uses this library to provide this packages, they are basically just dependencies of this library. (whereas cypress is obviously installed if you want to use this).

Reply:
You can also use [peerdependenciesmeta](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta) to define an optional PeerDep :)

Reply:
But again are that peer dependencies? I think not as that are plain dependencies.
Why should an app depend on `dockerode`, it should rather depend on `@nextcloud/cypress` which uses `dockerode` under the hood.